### ISSUE TITLE
Termination should also "discard" the receiving browsing context

### ISSUE THREAD

**Issue by tidoust** [w3c/presentation-api#423]
The steps to [terminate a presentation in a receiving browsing context](https://w3c.github.io/presentation-api/#dfn-terminate-a-presentation-in-a-receiving-browsing-context) tell the receiving user agent to [unload a document](http://www.w3.org/TR/html51/browsers.html#unloaded).

I believe the intent is to close the context entirely, but there are a couple of additional steps needed to [close a browsing context](http://www.w3.org/TR/html51/browsers.html#close-a-browsing-context):

- Remove the specified browsing context from the user interface (e.g., close or hide its tab in a tabbed browser). 
- [Discard](http://www.w3.org/TR/html51/browsers.html#a-browsing-context-is-discarded) the specified browsing context.

The first point is UA-specific and could perhaps go without saying, but the second point is neeed to set [`window.closed`](http://www.w3.org/TR/html51/browsers.html#dom-window-closed). A quick test suggests that Chrome already does that correctly.

In other words, I propose to revisit the bullet point we have to "If there is a receiving browsing context for P, and it has a document for P that is not unloaded, unload a document corresponding to that browsing context, remove that browsing context from the user interface and discard it".

**Comment by mfoltzgoogle** [comment]
SGTM
